env.example

```diff
@@ -48,6 +48,11 @@ export DEPLOY_VERSION=v3.11
 #export IMAGE_RESOURCEGROUP=images
 #export IMAGE_RESOURCENAME=$(az image list -g $IMAGE_RESOURCEGROUP -o json --query "[?starts_with(name, '${DEPLOY_OS:-rhel7}-${DEPLOY_VERSION//v}') && tags.valid=='true'].name | sort(@) | [-1]" | tr -d '"')
 
+# If building a custom image, uncomment the following lines and re-source your env
+#export IMAGE_RESOURCEGROUP="${USER}-images"
+#export IMAGE_RESOURCENAME="${DEPLOY_OS:-rhel7}-${DEPLOY_VERSION//v}-$(date -u +%Y%m%d%H%M)"
```

Comment:
s/${DEPLOY_OS:-rhel7}/rhel7/

(I have a separate PR in parallel removing DEPLOY_OS across the board)